doc/Requirements Specification/Parts/Non-Functional Requirements.lyx

```diff
@@ -241,8 +241,21 @@ name "/OQ30/"
 
 \end_inset
 
- If in the software which is measured, an error occur and e.g.
- the program is killed or throws exeptions, Beagle shall handle it.
+ Beagle shall handle any error thrown by the measured software, like uncaught
+ Exceptions or calls to 
+\begin_inset listings
+inline true
+status open
+
+\begin_layout Plain Layout
+
+System#exit
```

Comment:
If we run the program in its own thread, we can surely catch when this thread exits. We don’t need to prevent `System#exit`. We just need to make sure it does not kill Beagle.